views/route_segments/route_segments_show.html

```diff
@@ -70,6 +71,14 @@ <h2 class="{{ IconFor `route-segment` }}">{{ .Name }}</h2>
           </div>
         </div>
       </div>
+
+      <div class="inner-form h-[300px] md:h-[500px] print:hidden">
+        <h3>
+          <span>{{ i18n "Personal trend" }}</span>
```

Comment:
you should run `make translation`; this may fail because you need `xspreak` (which could then be added to `install-dev-deps` ...)

Reply:
```bash
go install github.com/vorlif/xspreak@latest
```